draft-irtf-nwcrg-coding-and-congestion-04.txt

```diff
@@ -425,11 +477,14 @@ Internet-Draft            Coding and congestion                July 2020
                      Figure 6: FEC below the transport
 
    Figure 6 presents an architecture where FEC is below the transport.
-   The repair packets are sent on top of what is allowed by the
+   The repair symbols are sent on top of what is allowed by the
    congestion control.
```

Comment:
I suggest to add, here: "Note that it is common to apply FEC at the link layer, in which it contributes to the total capacity that a link exposes to upper layers. This application of FEC is out of scope of this document."

Reply:
thanks